src/main/kotlin/no/nav/familie/ef/sak/infrastruktur/config/WebConfig.kt

```diff
@@ -19,6 +19,7 @@ class WebConfig(private val tilgangInterceptor: TilgangInterceptor) : WebMvcConf
             "/v2/api-docs/**",
             "/v2/api-docs",
             "/api/ekstern/perioder/full-overgangsstonad", // håndteres i controllern
+            "/api/ekstern/minside/stonadsperioder", // urelevant å sjekke roller for tokenx/kall fra min-side
```

Comment:
Se no.nav.familie.ef.sak.infrastruktur.sikkerhet.TilgangInterceptor

Før versjon 4 av no.nav.security returnerte getClaims() "null" hvis man ikke hadde noen getJwtTokenClaims for en issuer. Når vi prøvde å hente for tokenx returnerte vi et emptySet() med roller hvis vi fikk null fra SpringTokenValidationContextHolder().getTokenValidationContext().getClaims
(no.nav.familie.ef.sak.infrastruktur.sikkerhet.SikkerhetContext.hentGrupperFraToken - no.nav.security.token.support.core.context)

Nå kaster getClaims() IllegalArgumentException

Hvorfor ikke løse dette med å hente ut claims selv jwtToken(issuerName)?.jwtTokenClaims og beholde kode som den var?

Tokenx brukerne har ingen roller i ef-sak. En sideeffekt av at vi returnerte null var at vi fortsatte med å sjekke hentSaksbehandlerEllerSystembruker som ga oss SYSTEM_FORKORTELSE dersom det ikke var noen NAVident i azuread claim. Dette ble jo også litt feil.
Dette fungerte tidligere da

Løsning: exclude endepunkt for tokenx i. Vi sjekker fortsatt om bruker har et gyldig token. Vi trenger ikke sjekke roller i ef-sak

Reply:
Bør vi teste automatisk journalføring - da kaller jo mottak på en del ting uten "azuread" også?